runner/config.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	dftConf = ".air.conf"
+	dftConf = ".air.toml"
```

Comment:
We need to consider compatibility problem.if we accept this alter. and user update the air,will cause the air not work.

Reply:
@xiantang 
Fix to support both `.air.toml` and `.air.conf`!